binding/python/test_porcupine.py

```diff
@@ -73,7 +46,50 @@ def test_process_multiple(self):
 
         porcupine.delete()
 
-        self.assertEqual(results, [6, 0, 1, 2, 3, 4, 5, 6, 7])
+        self.assertEqual(results, ground_truth)   
+
+    def test_model_en(self):
+        lang = 'en'
+
+        self.run_porcupine(lang = lang,
+            audio_file_name = 'porcupine.wav',
+            keywords = ['porcupine'],
+            ground_truth = [0])        
+
+        self.run_porcupine(lang = lang,
+            audio_file_name = 'multiple_keywords.wav',
+            keywords = ['americano', 'blueberry', 'bumblebee', 'grapefruit', 'grasshopper', 'picovoice', 'porcupine', 'terminator'],
+            ground_truth = [6, 0, 1, 2, 3, 4, 5, 6, 7])
+
+    def test_model_es(self):
+        lang = 'es'
+
+        # this test failed due to utf8 problem in windows
+        # self.run_porcupine(lang = lang,
+        #     audio_file_name = 'multiple_keywords_es.wav',
+        #     keywords = ['emparedado', 'murciélago', 'leopardo', 'manzana'],
+        #     ground_truth = [0, 2, 3, 1])
```

Comment:
we don't generally merge in code that's commented out. If you are waiting on a merge from someone else, you can either wait for it to be merged and then rebase off their changes or put the changes on a new branch and merge it at a later time